app/javascript/components/shared_accesses/SharedAccessList.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import {
+  Button, Card, Col, Row, Stack,
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import Modal from '../shared/Modal';
+import SharedAccessForm from '../forms/shared_access_forms/SharedAccessForm';
+import DeleteSharedAccessForm from '../forms/shared_access_forms/DeleteSharedAccessForm';
+import Avatar from '../users/Avatar';
+import SearchBar from '../shared/SearchBar';
+
+export default function SharedAccessList({ users }) {
+  const [search, setSearch] = useState('');
+
+  return (
+    <div id="shared-access-list" className="wide-background full-height-room">
+      <Stack direction="horizontal" className="w-100 mt-5">
+        <SearchBar id="shared-users-search" setSearch={setSearch} />
+        <Modal
```

Comment:
You can add the border and shadow here through className

Reply:
I found we can pass a contentClassName prop to the Modal to access modal-content.
I put the Bootstrap classes directly on the template because I am assuming that all Modals will have the same design.
Do you agree? Or should we pass the Bootstrap as props every time we render a modal?

Reply:
No that's fine